help/sources/tutorials/ui/create/analytics/pendo-webhook.md

```diff
@@ -0,0 +1,165 @@
+
+* **[!UICONTROL Connection]**: Shows the source type, the relevant path of the chosen source file, and the amount of columns within that source file.
+* **[!UICONTROL Assign dataset & map fields]**: Shows which dataset the source data is being ingested into, including the schema that the dataset adheres to.
+
+Once you have reviewed your dataflow, click **[!UICONTROL Finish]** and allow some time for the dataflow to be created.
+
+![The review step of the sources workflow.](../../../../images/tutorials/create/analytics-pendo-webhook/review.png)
+
+## Get your streaming endpoint URL {#get-streaming-endpoint-url}
+
+With your streaming dataflow created, you can now retrieve your streaming endpoint URL. This endpoint will be used to subscribe to your webhook, allowing your streaming source to communicate with Experience Platform. 
+
+Next, you will require to construct a URL using the API details generated within Platform which will be used to configure the webhook on [!DNL Pendo].
+
+Note down the values of the fields below: 
+* **[!UICONTROL Dataflow ID]**
+* **[!UICONTROL Streaming endpoint]**
+
+To retrieve your **[!UICONTROL Dataflow ID]** and **[!UICONTROL Streaming endpoint]**, go to the [!UICONTROL Dataflow activity] page of the dataflow that you just created and copy the details from the bottom of the [!UICONTROL Properties] panel.
+
+![The streaming endpoint in dataflow activity.](../../../../images/tutorials/create/analytics-pendo-webhook/endpoint-test.png)
+
+Construct a URL using these values following the pattern below:
+```{STREAMING_ENDPOINT}?x-adobe-flow-id={DATAFLOW_ID}```
+
+Example of a constructed webhook URL: ```https://dcs.adobedc.net/collection/0c61859cc71939a0caf01123f91b2fc52589018800ad46b6c76c2dff3595ee95```
+
+## Next steps {#next-steps}
+
+By following this tutorial you have successfully [configured a dataflow to bring data into Platform](https:/experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/dataflow/crm.html).
+
+Next, login to your account on [[!DNL Pendo]](https:/pendo.io/) and create a webhook by following the [[!DNL Pendo] create a webhook](https://support.pendo.io/hc/en-us/articles/360032285012-Webhooks#create-a-webhook-0-4) tutorial. You need to create a URL Webhook.
+
+Navigate to the **[!UICONTROL Settings]** > **[!UICONTROL Integrations]** > **[!UICONTROL Webhook]** page and use the [URL](#get-streaming-endpoint-url) that was constructed, in the **[!UICONTROL URL]** field.
+![[DNL Pendo] UI screenshot showing the webhook endpoint field](../../../../images/tutorials/create/analytics-pendo-webhook/webhook.png)
+
+Refer to the [!DNL Pendo] [documentation](https://support.pendo.io/hc/en-us/articles/360032285012-Webhooks#create-a-webhook-0-4) to select individual event categories to choose which event categories you want to send into Platform.
```

Comment:
Like the other ones, lets put this into its own section between the step where endpoint is retrieved and next steps